web/test/pages/dashboard.test.tsx

```diff
@@ -668,33 +668,173 @@ describe("dashboard page", () => {
       }
     );
 
-    it.each(operatingPhasesWithQuickActions)(
-      "displays state-contracting-external-link quick action when industry id is not home-contractor for %s",
-      (phase) => {
-        useMockProfileData({ operatingPhase: phase, industryId: "some-value" });
-        renderDashboardPage({
-          quickActionLinks: [
-            generateQuickActionLink({
-              name: "some name",
-              filename: "state-contracting-external-link",
-              externalRoute: "some-external-route",
-            }),
-          ],
-        });
-        expect(screen.getByText("some name")).toBeInTheDocument();
-      }
-    );
+    describe("state-contracting-external-link", () => {
+      it.each(operatingPhasesWithQuickActions)(
+        "displays state-contracting-external-link quick action when business persona is oscar and sector id is not construction for %s",
+        (phase) => {
+          useMockProfileData({ operatingPhase: phase, sectorId: "some-value", businessPersona: "OWNING" });
+          renderDashboardPage({
+            quickActionLinks: [
+              generateQuickActionLink({
+                name: "some name",
+                filename: "state-contracting-external-link",
+                externalRoute: "some-external-route",
+              }),
```

Comment:
[dust] I think it might be interesting to toy around with this kind of paradigm (totally 100% optional though). You might have to wrap everything in a loop to capture the opteratingPhasesWithQuickActions but you could then control / represent the various sectorId's, businessPersona's and industryId's very cleanly. 
<img width="598" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/662086d2-63b1-4063-87ff-f41871bf375b">